kie-server-parent/kie-server-router/kie-server-router-proxy/src/main/java/org/kie/server/router/KieServerRouter.java

```diff
@@ -109,7 +128,38 @@ public KieServerRouter() {
         log.info("KIE Server router repository implementation is " + repository);
     }
 
-    public static void main(String[] args) {
+    public static void main(String[] args) throws Exception {
+        Options options = new Options();
```

Comment:
What is the purpose of this CLI adjustment?

Reply:
you need a way to add / remove instances from the identity manager...

Reply:
ok, thanks